src/index.ts

```diff
@@ -62,6 +63,7 @@ export const cmp = (window.guCmpHotFix.cmp ||= {
 	init,
 	willShowPrivacyMessage,
 	showPrivacyManager,
+	version: __PACKAGE_VERSION__,
```

Comment:
i think you'll need to declare this as a global

Reply:
TIL about globals in Jest, ESLint & TS: e511a63